Make docs more LLM friendly (#29566)
dagster-io/dagster: docs/docs/dagster-plus/deployment/deployment-types/serverless/dagster-ips.md

---

> description: Dagster+ Serverless IP addresses for outbound requests; whitelist these

Reviewer: ```suggestion
description: Add these Dagster+ Serverless IP addresses to an allowlist for outbound requests to external services.
```

>   for service interactions. 

Reviewer: ```suggestion
```

sidebar_label: IP addresses
sidebar_position: 400
title: Dagster+ Serverless IP addresses
---

> Serverless code will make requests from one of the following IP addresses. You may need to whitelist / allowlist them for services your code interacts with.

Reviewer: ```suggestion

Serverless code will make requests from one of the following IP addresses. You may need to whitelist / allowlist them for services your code interacts with.
```

Reviewer: (We want the extra space here)
